src/pages/_Components/Picker/Picker.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useState} from "react";
```

Comment:
Think this file could possibly do with a refactor, particularly around the optionMapper data structure.
But if we're eager to get this in we could look at that later.

Reply:
Will refactor Picker and TechSupportedByStacks components separate from the PR